{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"name":"Verifica_versão.ipynb","provenance":[],"collapsed_sections":[]},"kernelspec":{"name":"python3","display_name":"Python 3"}},"cells":[{"metadata":{"id":"gSHqMRhFzcAb"},"cell_type":"code","source":["import pandas\n","import numpy\n","import seaborn\n","import scipy"],"execution_count":null,"outputs":[]},{"metadata":{"id":"GQ5bxfxXzvKp","colab":{"base_uri":"https://localhost:8080/"},"outputId":"35f3e284-71c6-4e51-f1f9-8e4135a28e3e","executionInfo":{"status":"ok","timestamp":1652467328876,"user_tz":180,"elapsed":6,"user":{"displayName":"Tárik Pomim","userId":"17902036952921676844"}}},"cell_type":"code","source":["print('Versão do pandas -> %s' % pandas.__version__)\n","print('Versão do numpy -> %s' % numpy.__version__)\n","print('Versão do seaborn -> %s' % seaborn.__version__)\n","print('Versão do scipy -> %s' % scipy.__version__)"],"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["Versão do pandas -> 1.3.5\n","Versão do numpy -> 1.21.6\n","Versão do seaborn -> 0.11.2\n","Versão do scipy -> 1.4.1\n"]}]},{"cell_type":"code","source":["# voltando para uma versão anterior do pandas:"],"metadata":{"id":"GbNKWSY6u3Cm"},"execution_count":null,"outputs":[]},{"metadata":{"id":"IYakhWIx0Erv","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1652467511319,"user_tz":180,"elapsed":136481,"user":{"displayName":"Tárik Pomim","userId":"17902036952921676844"}},"outputId":"fc0eaa45-e7af-4fcc-966b-b3bd6cb9e67c"},"cell_type":"code","source":["!pip install pandas=='0.21.0'"],"execution_count":null,"outputs":[{"output_type":"stream","name":"stdout","text":["Collecting pandas==0.21.0\n","  Downloading pandas-0.21.0.tar.gz (11.3 MB)\n","\u001b[K     |████████████████████████████████| 11.3 MB 4.6 MB/s \n","\u001b[?25h  Installing build dependencies ... \u001b[?25l\u001b[?25herror\n","\u001b[33mWARNING: Discarding https://files.pythonhosted.org/packages/84/4c/622301f99725c6da76ae8079cb6b9091d9098098bd6b4eac47fb502b32f4/pandas-0.21.0.tar.gz#sha256=5cd5cb30e72eeaf202f0e5e180780b897570e889d2db328c689a5a263405c559 (from https://pypi.org/simple/pandas/). Command errored out with exit status 1: /usr/bin/python3 /tmp/pip-standalone-pip-43dc8g9t/__env_pip__.zip/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-3cc41m7h/overlay --no-warn-script-location -v --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- wheel setuptools Cython 'numpy==1.9.3; python_version=='\"'\"'3.5'\"'\"'' 'numpy==1.12.1; python_version=='\"'\"'3.6'\"'\"'' 'numpy==1.13.1; python_version>='\"'\"'3.7'\"'\"'' Check the logs for full command output.\u001b[0m\n","\u001b[31mERROR: Could not find a version that satisfies the requirement pandas==0.21.0 (from versions: 0.1, 0.2, 0.3.0, 0.4.0, 0.4.1, 0.4.2, 0.4.3, 0.5.0, 0.6.0, 0.6.1, 0.7.0, 0.7.1, 0.7.2, 0.7.3, 0.8.0, 0.8.1, 0.9.0, 0.9.1, 0.10.0, 0.10.1, 0.11.0, 0.12.0, 0.13.0, 0.13.1, 0.14.0, 0.14.1, 0.15.0, 0.15.1, 0.15.2, 0.16.0, 0.16.1, 0.16.2, 0.17.0, 0.17.1, 0.18.0, 0.18.1, 0.19.0, 0.19.1, 0.19.2, 0.20.0, 0.20.1, 0.20.2, 0.20.3, 0.21.0, 0.21.1, 0.22.0, 0.23.0, 0.23.1, 0.23.2, 0.23.3, 0.23.4, 0.24.0, 0.24.1, 0.24.2, 0.25.0, 0.25.1, 0.25.2, 0.25.3, 1.0.0, 1.0.1, 1.0.2, 1.0.3, 1.0.4, 1.0.5, 1.1.0, 1.1.1, 1.1.2, 1.1.3, 1.1.4, 1.1.5, 1.2.0, 1.2.1, 1.2.2, 1.2.3, 1.2.4, 1.2.5, 1.3.0, 1.3.1, 1.3.2, 1.3.3, 1.3.4, 1.3.5)\u001b[0m\n","\u001b[31mERROR: No matching distribution found for pandas==0.21.0\u001b[0m\n"]}]},{"cell_type":"code","source":[""],"metadata":{"id":"NQxn54ZFuxk8"},"execution_count":null,"outputs":[]}]}